src/onDraw/dataExport/exportToXLSX/defineReportXLSX/deriveReportData.js

```diff
@@ -21,10 +20,10 @@ export default function deriveReportData(id) {
     };
 
     //Define headers.
-    this.export.headers = d3.merge([this.export.nests, this.config.headers.slice(1)]);
+    this.export.headers = merge([this.export.nests, this.config.headers.slice(1)]);
```

Comment:
guessing headers includes ID?

Reply:
Exactly - couldn't have that messing up the party